zhaquirks/ikea/symfonisk2.py

```diff
@@ -0,0 +1,347 @@
+            COMMAND: COMMAND_MOVE,
+            CLUSTER_ID: 8,
+            ENDPOINT_ID: 1,
+            PARAMS: {"move_mode": 1},
+        },
+        (SHORT_PRESS, RIGHT): {
+            COMMAND: COMMAND_STEP,
+            CLUSTER_ID: 8,
+            ENDPOINT_ID: 1,
+            PARAMS: {"step_mode": 0},
+        },
+        (SHORT_PRESS, LEFT): {
+            COMMAND: COMMAND_STEP,
+            CLUSTER_ID: 8,
+            ENDPOINT_ID: 1,
+            PARAMS: {"step_mode": 1},
+        },
+    }
+
+
+class IkeaSYMFONISKgen2V1(CustomDevice):
+    """Custom device representing IKEA SYMFONISK sound remote gen2 V1.0.012."""
+
+    signature = {
+        # <SimpleDescriptor endpoint=1 profile=260 device_type=6
+        # device_version=1
+        # input_clusters=[0, 1, 3, 32, 4096, 64636]
+        # output_clusters=[3, 4, 6, 8, 25, 4096, 64639]>
+        MODELS_INFO: [(IKEA, "SYMFONISK sound remote gen2")],
+        ENDPOINTS: {
+            1: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.REMOTE_CONTROL,
+                INPUT_CLUSTERS: [
+                    Basic.cluster_id,
+                    PowerConfiguration.cluster_id,
+                    Identify.cluster_id,
+                    PollControl.cluster_id,
+                    LightLink.cluster_id,
+                    WWAH_CLUSTER_ID,
+                ],
+                OUTPUT_CLUSTERS: [
+                    Identify.cluster_id,
+                    Groups.cluster_id,
+                    OnOff.cluster_id,
+                    LevelControl.cluster_id,
+                    Ota.cluster_id,
+                    LightLink.cluster_id,
+                    ShortcutV1Cluster.cluster_id,
+                ],
+            },
+        },
+    }
+
+    replacement = {
+        ENDPOINTS: {
+            1: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: zha.DeviceType.REMOTE_CONTROL,
+                INPUT_CLUSTERS: [
+                    Basic.cluster_id,
+                    PowerConfiguration2AAACluster,
+                    Identify.cluster_id,
+                    PollControl.cluster_id,
+                    LightLink.cluster_id,
+                    WWAH_CLUSTER_ID,
+                    ShortcutV1Cluster,
+                ],
+                OUTPUT_CLUSTERS: [
+                    Identify.cluster_id,
+                    Groups.cluster_id,
+                    OnOff.cluster_id,
+                    LevelControl.cluster_id,
+                    Ota.cluster_id,
+                    LightLink.cluster_id,
+                    ShortcutV1Cluster,
+                ],
+            },
+        },
+    }
+
+    device_automation_triggers = (
+        Symfonisk2CommonTriggers.device_automation_triggers.copy()
+    )
+    device_automation_triggers.update(
+        {
+            (SHORT_PRESS, BUTTON_1): {
+                COMMAND: COMMAND_SHORTCUTV1,
+                PARAMS: {"Shortcut_Button": 1, "Shortcut_Event": 1},
```

Comment:
(Note: If the upper/lower-casing changes for the `server_commands`, it also needs to be changed here)